### ISSUE TITLE
[feature] Remote auth storage (mongo)

### ISSUE THREAD

**Issue by creatorrr** [allburov/whatsapp-http-api#41]

**Comment by allburov** [comment]
Hi, thank you for letting us know! We'll look at this!

For other who are interested in the feature - please hit thumbs up emoji in the above comment👍

**Comment by creatorrr** [comment]
Thanks @allburov ! Any news on this front? This would be a huge reason for us to get a plus subscription. :)

**Comment by allburov** [comment]
Hi! 
Sorry, no progress on it so far :( 

If you subscribe now - it'll help the project anyway and give us more time for the project and the feature!😉

**Comment by creatorrr** [comment]
Subscribed on patreon already :)

**Comment by nicolaslopez1000** [comment]
Hey, any update on this issue? Abt to develop it on my own

**Comment by allburov** [comment]
Hi! We're planning to implement it in September\October, hopefully!

**Comment by tarekkoura** [comment]
Any updates for chats on NOWEB engine?

**Comment by allburov** [comment]
Hi! It's the most important task for 2024, hopefully we'll add it soon!

**Comment by devlikepro** [comment]
Hi! 
We know that it's been more then a year since the request was requested, but we're happy to announce MongoDB remote storage for authentication data is available in `2023.4.2` version!

This feature is now available as part of the [WAHA Plus](https://waha.devlike.pro/docs/how-to/plus-version/) offering.

We have documented the feature and its usage in our documentation, available at [Session Storage](https://waha.devlike.pro/docs/how-to/storages/). 

----

To get started with the MongoDB storage, you can follow these simple steps:

First, start the MongoDB server by running the following command:
```bash
docker run -d -p 27017:27017 --name mongodb mongo
```

Next, initiate the WAHA with the `WHATSAPP_SESSIONS_MONGO_URL` environment variable, and note the use of the `--network host` option as well:
```bash
docker run -e WHATSAPP_SESSIONS_MONGO_URL=mongodb://localhost:27017/ --network host devlikeapro/whatsapp-http-api-plus
```

Following these steps, you will have successfully configured the MongoDB storage for WAHA. All session authentication data will now be stored in the MongoDB database.


For those of you eager to test this new feature, we would like to stress the importance of thoroughly testing MongoDB storage before deploying it in any production-related infrastructure. 


[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)
